Earthfile

```diff
@@ -604,10 +644,21 @@ iso:
 
 
 iso-uki:
+    FROM ubuntu
```

Comment:
why ubuntu? A smaller image like alpine is better suited for this, especially if its just to run a script?

Reply:
alpine had no `bash`:

```
~/workspace/kairos/kairos (1898-byoi)*$ docker run -it --rm alpine /bin/bash
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/bin/bash": stat /bin/bash: no such file or directory: unknown.
```

We could implement the script in pure `sh` but we opted for an image that has bash.